src/Components/v2/Carousel.tsx

```diff
@@ -388,7 +391,24 @@ export class BaseCarousel extends React.Component<
     }
 
     return (
-      <>
+      <ErrorBoundary
+        onCatch={() => {
+          /**
+           * If an error occurs when mounting / unmounting flickity, reload page.
+           *
+           * FIXME: Figure out how to diagnose `Failed to execute ‘removeChild’
+           * on ‘Node’: The node to be removed is not a child of this node.` error
+           *
+           * NOTE: To hack-fix error, apply a random `key={Math.random()} on
+           * offending container, or ensure component isn't rerendering needlessly.
+           */
+          console.warn(
+            "------------------ RELOAD carousel",
+            window.location.href
```

Comment:
Not actually doing what the comment says it's doing.

Reply:
It is I think.

> If an error occurs when mounting / unmounting flickity, reload page.

this is the `onCatch`, which does a reload.